packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -2231,19 +2231,6 @@ final class _RenderDeferredLayoutBox extends RenderProxyBox with _RenderTheaterM
     super.redepthChildren();
   }
 
-  bool _callingMarkParentNeedsLayout = false;
-  @override
-  void markParentNeedsLayout() {
```

Comment:
This override is mentioned in a comment in line 2191 as a crucial feature. Can you update that comment with the new findings?

Reply:
Updated that comment.